run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -o errexit
 
-if [[ ${OSH_DEVELOPER_MODE:-"False"} == "True" ]]; then
+if [[ ${SOCOK8S_DEVELOPER_MODE:-"False"} == "True" ]]; then
```

Comment:
good catch. You might also want to wrap the left hand side of the `==` with ```""```

Reply:
Wrap 'SOCOK8S_DEVELOPER_MODE' ? e.g. 
${"SOCOK8S_DEVELOPER_MODE":-"False"}. 

I see this can be a post-merge cleanup / improvement action if its okay.

Reply:
``"${SOCOK8S_DEVELOPER_MODE:-False}" `` is what I meant. I am afraid of false positives.

Reply:
But it can be done in a different commit if you prefer.

Reply:
Thanks. Will make this change in next patch